core/object/script_language.h

```diff
@@ -429,6 +429,8 @@ class ScriptLanguage : public Object {
 	virtual bool refcount_decremented_instance_binding(Object *p_object) { return true; } //return true if it can die //optional, not used by all languages
 
 	virtual void frame();
+	virtual void set_physics_process_delta_time(double p_time);
+	virtual void set_process_delta_time(double p_time);
```

Comment:
I thought of using the `process_frame` and `physics_frame` signals, but our callback needs to have priority, as we want the delta time to be updated before the other callbacks are called.